modules/nw-ingress-setting-select-subnet-LoadBalancerService.adoc

```diff
@@ -0,0 +1,89 @@
+// Modules included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-setting-select-subnet-LoadBalancerService_{context}"]
+= Choosing subnets while creating a LoadBalancerService Ingress Controller
+You can manually specify load balancer subnets for Ingress Controllers in an existing cluster. By default, the load balancer subnets are automatically discovered by AWS, but specifying them in the Ingress Controller overrides this, allowing for manual control.
+
+.Prerequisites
+* You must have an installed AWS cluster.
+* You must know the names or IDs of the subnets to which you intend to map your `IngressController`
+
+.Procedure
+To create an `IngressController` with manually specified load balancer subnets, you can follow these steps:
+
+. Create a custom resource (CR) file.
++
+Create a YAML file (e.g., `sample-ingress.yaml`) with the following content:
```

Comment:
Not really sure you need the " (e.g., `sample-ingress.yaml`)" here, but if you do, should there be two names to differentiate the CR created in step 1 vs step 2?